docs/source/acquire.rst

```diff
@@ -0,0 +1,31 @@
+acquire
```

Comment:
Maybe also nice to explain that other target-* commands can be executed on the .tar file. For someone who is new to Dissect it might not be that straight forward to think that can be done.

Reply:
Added output decsription.